tests/unit/test_reporting.py

```diff
@@ -415,15 +414,15 @@ def test_report_status_by_comment(
             GithubCheckRunResult.success,
             CommitStatus.success,
             (
-                github.GithubException,
+                GithubAPIException,
                 # https://docs.python.org/3/library/inspect.html#inspect.signature
                 # to account for changes in positional arguments: pygithub 1.55 added headers
                 # as additional positional argument; this creates an iterable and sets None
-                # for every argument of GithubException.__init__ except for 'self'
+                # for every argument of GithubAPIException.__init__ except for 'self'
                 [
                     None
                     for param_name, param in signature(
-                        github.GithubException.__init__
+                        GithubAPIException.__init__
```

Comment:
same here…

looking at it again, it seems like it is some kind of a workaround for different versions of PyGithub? IMO it's no longer relevant :shrug: but I might be wrong :D